.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
nit; its standard to have the `.vscode` directory in your `.gitignore` file and not committed.

Reply:
Really? I thought it was specifically a way to share project settings between members, e.g. so we don't differ on final newline.

Reply:
One thing I could do is ignore `.vscode` but allow these specific files.

Reply:
Typically, we don't commit any of this stuff and have a team standard that everyone is expected to follow across all repos rather than potentially divergent configurations across multiple repositories. 

Like I said though, its a nit and you can do whatever you'd like with your repo.